.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
```

Comment:
we can pass daily, then it updates "faster". Example: like this, if a update is available on wednesday, it will only apply for us (open a PR bumping the github action that has an available update) on sunday. If we pass daily, it will update on wednesday

Reply:
I will change to daily. In the future we can change for weekly